charts/csi-baremetal-deployment/values.yaml

```diff
@@ -154,3 +154,5 @@ nodeController:
   metrics:
     path: /metrics
     port: 8787
+
+platform: ""
```

Comment:
Should we set default value? And add new line at the end of file please

Reply:
platform field should be set on non-vanilla setups otherwise use should empty value from defaults.